modules/microshift-install-rhel-image-mode-run-container.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhel-image-mode-run-container_{context}"]
+= Running the {microshift-short} bootc container
+
+If the host is already configured to have a `rhel` volume group with free space, this configuration is inherited by the container. It can be used by the {microshift-short} LVMS Container Storage Interface (CSI) to allocate storage.
+
+.Prerequisites
+
+* A {op-system-base-full} 9.4 host with an active Red Hat subscription for building {microshift-short} bootc images and running containers.
+* You are logged into the {op-system} 9.4 host using the user credentials that have `sudo` permissions.
+* You have a pull secret file for downloading the required {microshift-short} container images.
+* You configured host container storage for reusing available container images.
+
+.Procedure
+
+. Run the container for the {microshift-short} service by entering the following command:
++
+[source,terminal]
+----
+PULL_SECRET=~/.pull-secret.json
+IMAGE_NAME=microshift-4.16-bootc
+
+$ sudo podman run --rm -it --privileged \
+    -v "${PULL_SECRET}":/etc/crio/openshift-pull-secret:ro \
+    -v /var/lib/containers/storage:/var/lib/containers/storage \
+    --name "${IMAGE_NAME}" \
+    "${IMAGE_NAME}"
+----
+//moved the hotwiring openvswitch over to CNI module
+
+[NOTE]
+====
+The `systemd-modules-load` service fails to start in the container if the host kernel version is different from the bootc image kernel version. This failure can be safely ignored as all the necessary kernel modules have been loaded by the host.
+====
+
+A login prompt like the following is presented in the terminal after {microshift-short} has started:
++
+[source,terminal]
+----
+$ tbd example
```

Comment:
Well, it may look like this, but I'm not sure we should be displaying it.

```
Red Hat Enterprise Linux 9.20240820.0.4 (Plow)
Kernel 5.14.0-427.28.1.el9_4.x86_64 on an x86_64

5b2a7019fdf9 login: 
```

Reply:
That's fair. Removed that and just stated what should happen as part of the next step.